src/coreclr/jit/emitarm64.cpp

```diff
@@ -6493,6 +6507,129 @@ void emitter::emitIns_R_R_R(
     appendToCurIG(id);
 }
 
+/*****************************************************************************
+ *
+ *  Add an instruction storing 2 registers into a memory (pointed by reg3) and the offset
+    (immediate).
+ */
+
+void emitter::emitIns_SS_R_R_R_I(instruction ins,
```

Comment:
Why is this not a variant or extension of `emitIns_S_S_R_R`?

In that case, no `reg3` is passed because it is determined based on the lclvar. In this case, there are 2 lclvar. Presumably they would both have the same `FPbased` value, thus determining the same address base register. In fact, you could probably determine / assert the offsets are correct.

Reply:
I have referred the code from `emitIns_R_R_R_I()` instead which operates on vector registers as well. For example, it operates on `scale` of 2, 3 and 4, whereas `emitIns_S_S_R_R()` operates just on `scale == 3`. Also, it has special handling for `useRegForAdr` which I assume is just for the stack stores. And yes, the `reg3` is always `FP` or `SP`.